### DISCUSSION TITLE
antv x6 浏览器兼容性支持的最低浏览器版本是什么（chrome和firefox）

### DISCUSSION THREAD

**Discussion by hshuaibin** [antvis/X6#1728] [Q&A]
![image](https://user-images.githubusercontent.com/45307029/148317360-f4444b84-abfe-4bba-a0ea-31d1019c98c9.png)
官方文档中描述为支持较新版本，但是不知道具体到哪个版本不支持，我们发现firefox52.3版本浏览器打开页面为空白页面，是否是x6不支持该版本的浏览器，有没有一个具体的浏览器兼容性支持最低版本

**Comment by NewByVector** [comment]
没有针对各类型各版本浏览器进行测试，可以在你们需要支持的最低版本上进行测试一波。

**Reply by Yeti-xxx** [reply]
现在有相关的支持清单吗？
